cmd/rotate_certs.go

```diff
@@ -136,7 +136,7 @@ func (rcc *rotateCertsCmd) run(cmd *cobra.Command, args []string) error {
 		if rcc.containerService.Properties.MasterProfile != nil {
 			rcc.outputDirectory = path.Join("_output", rcc.containerService.Properties.MasterProfile.DNSPrefix)
 		} else {
-			rcc.outputDirectory = path.Join("_output", rcc.containerService.Properties.HostedMasterProfile.DNSPrefix)
+			return errors.New("can't determine output directory from nil MasterProfile")
```

Comment:
Can we add a unit test for this error path to `rotate_certs_test.go`? Or at least put a pin in it by creating a GitHub issue.

Reply:
Actually I'm going to punt on this one as this whole thing is going to make @jadarsie's life harder and so why not make it even worse.

(mostly) kidding... but tl;dr I'll work with him on rebasing once this lands and then adapting his rewrite of rotate certs to deal with this stuff.

Reply:
Added a thumbs down for visual fidelity